ts/features/payments/payment/store/reducers/index.ts

```diff
@@ -97,6 +97,12 @@ const reducer = (
         ...state,
         rptId: action.payload,
         paymentDetails: pot.toLoading(state.paymentDetails)
+        /*
+        tentativeByRptId: {
+          ...state.tentativeByRptId,
+          [action.payload]: (state.tentativeByRptId[action.payload] || 0) + 1
+        }
+        */
```

Comment:
I'm guessing this wasn't intended, right?

Reply:
Right! https://github.com/pagopa/io-app/pull/5401/commits/135eac7c13def489bdba6e775b4081a24e367d72